server/services/store/sqlstore/public_methods.go

```diff
@@ -139,7 +140,7 @@ func (s *SQLStore) CreateSubscription(sub *model.Subscription) (*model.Subscript
 }
 
 func (s *SQLStore) CreateUser(user *model.User) (*model.User, error) {
```

Comment:
Why not simply delete these function where you are directly returning an error?

Reply:
Deleted.